web/src/routes/System.jsx

```diff
@@ -343,6 +344,7 @@ export default function System() {
                           <Th>FPS</Th>
                           <Th>CPU %</Th>
                           <Th>Memory %</Th>
+                          <Th>Bandwidth KB/s</Th>
```

Comment:
```suggestion
                          <Th>Network Bandwidth</Th>
```

Reply:
I think this makes it more clear the type of bandwidth, and the unit is shown for each row already